Library/Core/StoryboardSegueIdentifierProtocol.swift

```diff
@@ -44,7 +44,7 @@ public struct StoryboardSegueIdentifier<Segue, Source, Destination>: StoryboardS
   }
 
   /// Create a new StoryboardSegue based on the identifier and source view controller
-  public func storyboardSegueWithSource(sourceViewController: Source)
+  public func storyboardSegue(withSource sourceViewController: Source)
```

Comment:
Maybe the argument label should be `withSourceViewController`?

This is a bit of a weird one.
Normally in Swift 3 types aren't named in argument labels. But "ViewController" here isn't the name of the type, but of the property.